src/services/itemMembership/repository.ts

```diff
@@ -683,17 +683,26 @@ export class ItemMembershipRepository extends MutableRepository<
   // UTILS
 
   /**
+   * Get all memberships related to the the given item's ancestors,
+   * and return the memberships we need to create so the accounts that have permissions higher in the tree
+   * keeps their best permission on the item after it is moved to the root.
+   *
```

Comment:
```suggestion
   * Retrieves all memberships related to the ancestors of the given item and returns the memberships we need to create so that accounts with permissions further up the tree maintain their best permissions on the item after it has been moved to the root.
```